plugins/pulp_rpm/plugins/distributors/iso_distributor/distributor.py

```diff
@@ -61,18 +61,19 @@ def publish_repo(self, transfer_repo, publish_conduit, config):
         repo = transfer_repo.repo_obj
         return super(ISODistributor, self).publish_repo(repo, publish_conduit, config)
 
-    def unpublish_repo(self, repo, config):
+    def unpublish_repo(self, transfer_repo, config):
```

Comment:
Also, this makes `unpublish_repo` consistent with `publish_repo` directly above it.